gengo/gengo.py

```diff
@@ -248,7 +249,7 @@ def get(self, **kwargs):
             # best to familiarize yourself  with the Gengo authentication
             # API. (http://developers.gengo.com/)
             query_params = dict([k, quote(str(v).encode('utf-8'))] for k, v
-                                in kwargs.items())
+                                in list(kwargs.items()))
```

Comment:
I think we don't need to call `list()` (Also another `list()`). If you think we need to call that, could you tell me why?